package.json

```diff
@@ -22,28 +22,27 @@
   "license": "MIT",
   "peerDependencies": {},
   "dependencies": {
-    "@expo/react-native-touchable-native-feedback-safe": "^1.1.0",
+    "@expo/react-native-touchable-native-feedback-safe": "^1.1.2",
     "clamp": "^1.0.1",
-    "core-decorators": "^0.12.2",
-    "fbemitter": "^2.0.2",
-    "fbjs": "^0.8.3",
-    "hoist-non-react-statics": "^1.0.5",
-    "invariant": "^2.2.1",
-    "lodash": "^4.6.1",
+    "core-decorators": "^0.17.0",
+    "fbemitter": "^2.1.1",
+    "fbjs": "^0.8.12",
+    "hoist-non-react-statics": "^1.2.0",
+    "invariant": "^2.2.2",
+    "lodash": "^4.17.4",
     "mirror-creator": "^1.1.0",
-    "react-addons-shallow-compare": "15.3.2",
     "react-clone-referenced-element": "^1.0.1",
     "react-mixin": "^3.0.5",
-    "react-native-drawer-layout-polyfill": "^1.1.0",
-    "react-native-tab-view": "0.0.52",
-    "react-redux": "^4.4.5",
-    "react-static-container": "^1.0.0",
+    "react-native-drawer-layout-polyfill": "^1.3.0",
+    "react-native-tab-view": "0.0.58",
+    "react-redux": "^5.0.3",
+    "react-static-container": "^1.0.1",
     "react-timer-mixin": "^0.13.3",
-    "redux": "^3.5.2",
-    "redux-batched-actions": "^0.1.2",
-    "reselect": "^2.5.1",
+    "redux": "^3.6.0",
+    "redux-batched-actions": "^0.1.5",
+    "reselect": "^3.0.0",
```

Comment:
I couldn't point out anything major but @skevy @janicduplessis can you guys confirm?